docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -217,6 +217,10 @@ connector.
       - A decimal value in the range (0, 1] used as a minimum for weights assigned to each split. A low value may improve performance
         on tables with small files. A higher value may improve performance for queries with highly skewed aggregations or joins.
       - 0.05
+    * - ``parquet.max-read-block-row-count``
+      - Not required. Sets the maximum number of rows read in a batch. For
```

Comment:
Remove the optimal performance stuff.. thats wrong or at least undetermined in the Trino context. We could potentially add something like "Experts only. Dont touch"